### ISSUE TITLE
Are you still interested in help with PHP?

### ISSUE THREAD

**Issue by manngo** [phallguy/refiddle-com#18]

**Comment by phallguy** [comment]
Yes I would like to get other runners setup on the tool. I just haven't had the time to actually get up to speed with the various stacks and get them set up securely on the refiddle server.

**Comment by manngo** [comment]
I have some skills in PHP, and would be happy to help if I can.

**Comment by phallguy** [comment]
That'd be great. Refiddle was designed to support remote regex runners. We have a PR for PHP but I was never free to get it setup. If you could get the runner's running on Heroku I could point refiddle to run PHP against those servers.

**Comment by phallguy** [comment]
Sorry, meant to reference PR #15
